apps/crn-server/src/data-providers/email-notification-service.ts

```diff
@@ -78,11 +78,10 @@ export class EmailNotificationService {
   async sendEmailNotification(
     action: EmailTriggerAction,
     manuscriptId: string,
-    flagEnabled: boolean,
     emailList: string,
     discussionId?: string,
   ): Promise<void> {
-    if (!flagEnabled && !emailList) return;
+    const isProduction = environmentName === 'production';
```

Comment:
in the lambda function the ENVIRONMENT is set to `production`
<img width="1965" alt="environment-name" src="https://github.com/user-attachments/assets/e9675ed4-212f-4a4f-a21d-ad3eb9dce4c4" />